Assets/Scripts/KillBallOnFall.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KillBallOnFall : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+
+    // Update is called once per frame
+    void Update()
+    {
+        if(transform.position.y < -3)
```

Comment:
don't hardcode, make -3 a public variable for people to set in the inspector, call it something like "dealthY"